css-nav-1/Overview.bs

```diff
@@ -1236,37 +1236,69 @@ follow the following steps:
-            find the points <var>P1</var> inside the <a>boundary box</a> of <var>searchOrigin</var>
-            and <var>P2</var> inside the <a>boundary box</a> of <var>candidate</var>
-            that minimize the <var>distance</var> between these two points,
-            when <var>distance</var> is defined as follows:
-
-            <dl>
-              <dt><var>distance</var>:
-                <dd><var>A</var> + <var>B</var> + <var>C</var> - <var>D</var>
-
-              <dt><var>A</var>:
-                <dd>The euclidian distance between <var>P1</var> and <var>P2</var>
-
-              <dt><var>B</var>:
-                <dd>The absolute distance in the <var>dir</var> direction between <var>P1</var> and <var>P2</var>
-
-              <dt><var>C</var>:
-                <dd>The absolute distance in the direction which is orthogonal to <var>dir</var> between <var>P1</var> and <var>P2</var>
-
-              <dt><var>D</var>:
-                <dd>The square root of the area of intersection between the <a>boundary boxes</a> of <var>candidate</var> and <var>searchOrigin</var>
-            </dl>
-        3. Return the item of the <var>candidates</var> set that has the smallest <var>distance</var>.
+            1. If the <var>searchOrigin</var> is an element, then find the points <var>P1</var> inside the <a>boundary box</a> 
+                of <var>searchOrigin</var> and P2 inside the boundary box of <var>candidate</var>.
+            2. Else if the <var>searchOrigin</var> is a point, then let the point <var>P1</var> be the <var>searchOrigin</var> 
+                and find the point <var>P2</var> inside the <a>boundary box</a>.
+        3. Let <var>intersectRect</var> is intersected area between a candidate and the search origin.
+        4. Select the <var>P1</var> and <var>P2</var> that make the smallest <var>distance</var> between those two points by 
+            <a>calculating the distance</a>.        
+        5. Return the item of the <var>candidates</var> set that has the smallest <var>distance</var>.
             If several have the same distance,
             return the first one in document order,
             unless its [=boundary box=] overlaps with the [=boundary box=] of an other item at the same distance,
             and that item is higher in the CSS [=painting order=].
             In that case, return that item instead,
-            unless it too is overlapped with another higher item at the same distance, recursively.
+            unless it too is overlapped with another higher item at the same distance, recursively.    
+</div>
+
+<div algorithm="to calculate the distance">
+To <dfn lt="calculating the distance | to calculate the distance">calculate the <var>distance</var></dfn>, the function is defined as below:
+
+<pre class="prod">
+    <var>distance</var> = <var>euclidian</var> - <var>alignment</var> + <var>displacement</var> - <var>sqrt(Overlap)</var>
+</pre>
+
+The meaning of each factor in the function is as follows:
```

Comment:
each factor -> each factors

Reply:
'each factor' is the correct grammar.

Reply:
I was a bit confused. Totally agree on it.